src/openforms/emails/tests/test_tasks.py

```diff
@@ -399,3 +435,201 @@ def test_no_email_sent_when_brk_congiguration_is_invalid_but_unused(self):
             send_email_digest()
 
         self.assertEqual(0, len(mail.outbox))
+
+    def test_invalid_general_registration_backend_is_sent(self):
+        form = FormFactory.create()
+        FormRegistrationBackendFactory.create(
+            form=form,
+            key="plugin1",
+            backend="test-invalid-backend",
+        )
+
+        with patch(
+            "openforms.emails.tasks.GlobalConfiguration.get_solo",
+            return_value=GlobalConfiguration(
+                recipients_email_digest=["user@example.com"]
+            ),
+        ):
+            send_email_digest()
+
+        sent_email = mail.outbox[0]
+
+        self.assertEqual(
+            sent_email.subject, "[Open Forms] Daily summary of detected problems"
+        )
+        self.assertEqual(sent_email.recipients(), ["user@example.com"])
+        self.assertIn(
+            _("The configuration for plugin 'Invalid backend' is invalid."),
```

Comment:
as remarked in another PR, it doesn't make a lot of sense to mark these strings translatable.